src/components/MDXContainer.js

```diff
@@ -16,6 +16,7 @@ import { DocTile, DocTiles } from './DocTile';
 import LandingPageHero from './LandingPageHero';
 import LandingPageTile from './LandingPageTile';
 import LandingPageTileGrid from './LandingPageTileGrid';
+import InlineSignup from './InlineSignup';
```

Comment:
this need to be `WrappedInlineSignup`?

Reply:
`WrappedInlineSignup` is the default export in InlineSignup.js so this should be fine. i don't think the build should actually be failing, but i'm going to cut another theme release to remove the InlineSignup component there. once i bump the theme here again i think that should fix the build failure